### ISSUE TITLE
Cannot load particle video

### ISSUE THREAD

**Issue by Chibuikem709** [soft-matter/pims#327]
Hello!

I am new to this so please excuse me if this issue has already been reported and solved:
I am woring through the trackpy tutorial; however, I am using my own particle data. I am trying to load a video file (format: .mov) via pims.Video using the command

 `v = pims.Video('/Users/chibuikemnwizu/Dropbox (Brown)/Research/9 1-100 2018-08-02 12-07-22 (Converted).mov')`

I would like to see the basic properties of the video so I simply execute:

`v`

I get the following error:
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
/anaconda3/lib/python3.7/site-packages/IPython/core/formatters.py in __call__(self, obj)
    700                 type_pprinters=self.type_printers,
    701                 deferred_pprinters=self.deferred_printers)
--> 702             printer.pretty(obj)
    703             printer.flush()
    704             return stream.getvalue()

/anaconda3/lib/python3.7/site-packages/IPython/lib/pretty.py in pretty(self, obj)
    400                         if cls is not object \
    401                                 and callable(cls.__dict__.get('__repr__')):
--> 402                             return _repr_pprint(obj, self, cycle)
    403 
    404             return _default_pprint(obj, self, cycle)

/anaconda3/lib/python3.7/site-packages/IPython/lib/pretty.py in _repr_pprint(obj, p, cycle)
    695     """A pprint that just redirects to the normal repr function."""
    696     # Find newlines and replace them with p.break_()
--> 697     output = repr(obj)
    698     for idx,output_line in enumerate(output.splitlines()):
    699         if idx:

/anaconda3/lib/python3.7/site-packages/pims/base_frames.py in __repr__(self)
    174 Pixel Datatype: {dtype}""".format(w=self.frame_shape[0],
    175                                   h=self.frame_shape[1],
--> 176                                   count=len(self),
    177                                   dtype=self.pixel_type)
    178 

TypeError: 'float' object cannot be interpreted as an integer

```

How can I solve this issue? I think the video is not loading properly and thus I cannot move on through the other steps without resolving this step first.

**Comment by caspervdw** [comment]
Hi, thanks for the report. Apparently the representation of the movie isn’t working. This is however a minor issue: you can probably still use the video in subsequent calls. Try for example `len(v)` to display the number of frames.

**Comment by MuadDev** [comment]
I see the same behavior as @Chibuikem709 does

**Comment by iMhack** [comment]
I see the same behavior as @Chibuikem709 does too

**Comment by iMhack** [comment]
You might try to use v._len, check if it's written inf

**Comment by caspervdw** [comment]
This is a bug, the `__len__` attribute should catch the float

**Comment by zoeith** [comment]
For me, this was fixed installing from active development (5a1cbc84c1dccf3f4f4fa5f464b881071ac2652f)
```
pip install https://github.com/soft-matter/pims/archive/master.zip
```
